letsencrypt/client.py

```diff
@@ -125,6 +130,28 @@ def register(config, account_storage, tos_cb=None):
 
     return acc, acme
 
+def perform_registration(acme, config):
+    """
+    Actually register new account, trying repeatedly if there are email
+    problems
+
+    :param .IConfig config: Client configuration.
+    :param acme.client.Client client: ACME client object.
+
+    :returns: Registration Resource.
+    :rtype: `acme.messages.RegistrationResource`
+
+    :raises .UnexpectedUpdate:
+    """
+    try:
+        return acme.register(messages.NewRegistration.from_data(email=config.email))
+    except messages.Error, e:
```

Comment:
Are you sure?  I tried to implement some [here](https://github.com/letsencrypt/letsencrypt/pull/1524/files#diff-eedefe06400da5ee1616093685c1b14aR57).